Classes/Domain/Model/Index.php

```diff
@@ -29,39 +29,45 @@ class Index
      * @see http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/indices-update-settings.html
      */
     static protected $updatableSettings = [
-        'index.number_of_replicas',
-        'index.auto_expand_replicas',
-        'index.blocks.read_only',
-        'index.blocks.read',
-        'index.blocks.write',
-        'index.blocks.metadata',
-        'index.refresh_interval',
-        'index.index_concurrency',
-        'index.codec',
-        'index.codec.bloom.load',
-        'index.fail_on_merge_failure',
-        'index.translog.flush_threshold_ops',
-        'index.translog.flush_threshold_size',
-        'index.translog.flush_threshold_period',
-        'index.translog.disable_flush',
-        'index.cache.filter.max_size',
-        'index.cache.filter.expire',
-        'index.gateway.snapshot_interval',
-        'index.routing.allocation.include',
-        'index.routing.allocation.exclude',
-        'index.routing.allocation.require',
-        'index.routing.allocation.disable_allocation',
-        'index.routing.allocation.disable_new_allocation',
-        'index.routing.allocation.disable_replica_allocation',
-        'index.routing.allocation.enable',
-        'index.routing.allocation.total_shards_per_node',
-        'index.recovery.initial_shards',
-        'index.gc_deletes',
-        'index.ttl.disable_purge',
-        'index.translog.fs.type',
-        'index.compound_format',
-        'index.compound_on_flush',
-        'index.warmer.enabled',
+        'settings.index.number_of_replicas',
```

Comment:
Interestingly that was broken since ever? The `settings` part of the path was just missing. See documentation about the possible keys for both requests.

Reply:
Does this really work like that now?

Those keys are passed to `PUT /_settings` directly in `updateSettings()` AFAICS, and according to https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-update-settings.html they do **not** have `settings.` as a prefix.

I guess I see the issue, IMO the `settings.` needs to be stripped for building the payload to be sent to Elasticsearch…

Reply:
Oh, well yes, that might be the other option, I will chck the docs and adjust it the other wy around then, but as it is now it won't do anything.

Reply:
right, will adapt, I see the docs.